src/modules/list.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-restricted-globals */
+import Cruds from '../cruds/curds.js';
+
+const task = new Cruds();
+const tasks = () => {
+  const dataTask = task.getSortedTasks();
+  const ul = document.getElementById('ul');
+  ul.innerHTML = '';
+  dataTask.forEach((listItem) => {
+    const li = document.createElement('li');
+    li.id = listItem.index;
+    ul.appendChild(li);
+    const div = document.createElement('div');
+    li.appendChild(div);
+    const input = document.createElement('input');
+    input.type = 'checkbox';
+    input.id = 'checkbox';
+    input.className = 'checkbox';
+    input.addEventListener('change', (event) => {
+      const index = task.tasks.findIndex((t) => listItem.index === t.index);
+      if (index === -1) {
+        return;
+      }
+      task.tasks[index].completed = event.target.checked;
+      task.editTask(listItem, listItem.description);
+      if (listItem.completed) {
+        input.setAttribute('checked', 'checked');
+        desc.style.textDecoration = 'line-through';
+        desc.style.color = '#A3A3A3';
+        document.getElementById('clear-all').style.textDecoration = 'underline';
+      } else {
+        input.setAttribute('checked', 'unchecked');
+        desc.style.textDecoration = 'none';
+        desc.style.color = '#000';
+      }
+    });
+    document.getElementById('clear-all').addEventListener('click', () => {
+      task.clearCompletedTasks();
+      location.reload();
```

Comment:
I notice the use of location.reload() in order to update the localStorage. This is optional but I would recommend using localStorage.setItem() here to store the updated version of the list.

Reply:
Thanks for this change I need to remove the location.reload(). I have done these changes because localStorage.setItem() is working properly.